RetsTalk/RetsTalk/Retrospect/Model/RetrospectManager.swift

```diff
@@ -9,41 +9,124 @@ import Foundation
 import Combine
 
 final class RetrospectManager: RetrospectManageable {
-    private(set) var retrospects: [Retrospect] = []
-    private(set) var retrospectsSubject: CurrentValueSubject<[Retrospect], Never>
     private let userID: UUID
+    private var retrospects: [Retrospect] {
+        didSet { retrospectsSubject.send(retrospects) }
+    }
+    private(set) var retrospectsSubject: CurrentValueSubject<[Retrospect], Never>
+    private let retrospectStorage: Persistable
+    private let assistantMessageProvider: AssistantMessageProvidable
     
-    init(userID: UUID) {
+    init(
+        userID: UUID,
+        retrospectStorage: Persistable,
+        assistantMessageProvider: AssistantMessageProvidable
+    ) {
         self.userID = userID
+        self.retrospects = []
         self.retrospectsSubject = CurrentValueSubject(retrospects)
+        self.retrospectStorage = retrospectStorage
+        self.assistantMessageProvider = assistantMessageProvider
     }
     
-    func fetchRetrospects(offset: Int, amount: Int) {
+    func fetchRetrospects(offset: Int, amount: Int) async throws {
+        let finishedRecentRequest = finishedRecentRetrospectFetchRequest(offset: offset, amount: amount)
+        let finishedRecentEntities = try await retrospectStorage.fetch(by: finishedRecentRequest)
+        retrospects.append(contentsOf: finishedRecentEntities)
+    }
+    
+    func fetchinitRetrospects(offset: Int, amount: Int) async throws {
```

Comment:
```suggestion
    func fetchInitRetrospects(offset: Int, amount: Int) async throws {
```
init을 소문자로 하신 이유가 있을까요??

Reply:
아.. 이건 init이 습관이 되다보니까 Init을 안하게 된 것 같네요 감사합니다!